src/main/java/ayds/spotisong/aurora/otherdetails/fulllogic/model/repository/ArtistRepositoryImp.kt

```diff
@@ -0,0 +1,35 @@
+package ayds.spotisong.aurora.otherdetails.fulllogic.model.repository
+
+import ayds.spotisong.aurora.otherdetails.fulllogic.model.Artist
+import ayds.spotisong.aurora.otherdetails.fulllogic.model.repository.external.ExternalService
+import ayds.spotisong.aurora.otherdetails.fulllogic.model.repository.local.LocalDB
+import ayds.spotisong.aurora.song.model.Song
+
+internal class ArtistRepositoryImp (val localDB: LocalDB, val externalService: ExternalService) : ArtistRepository {
```

Comment:
las propiedades deberian ser private
```suggestion
internal class ArtistRepositoryImp (private val localDB: LocalDB, private val externalService: ExternalService) : ArtistRepository {
```